### DISCUSSION TITLE
Question about displaying rack utilization

### DISCUSSION THREAD

**Discussion by Sivolen** [netbox-community/netbox#11270] [Q&A]
I noticed such a feature, the rack load percentage is correct, but the scale itself is somehow large
![raks](https://user-images.githubusercontent.com/7619656/208813243-c4639391-d6f5-436c-9d10-df1e9a3e9f10.jpg)

**Comment by kkthxbye-code** [comment]
No mention of what version you are using, no mention of where you are looking and no mention of if you are using any plugins. So all I can tell you is that it works on my instance and it works on the demo:

https://demo.netbox.dev/dcim/racks/

**Reply by Sivolen** [reply]
I have netbox (v3.4.1) python version 3.8 installed in the server 20.04, there are no additional plugins, I check in firefox 106.0.1 (64 bits).
The demo site also displays

**Reply by kkthxbye-code** [reply]
Demo site in firefox 108:

![image](https://user-images.githubusercontent.com/400797/208838335-91bd1707-97f5-44db-bb89-12e75c2f0bad.png)

Works exactly as intended.

**Reply by Sivolen** [reply]
Hi!
I have it displayed like this ) 
![Screenshot 2022-12-21 174327](https://user-images.githubusercontent.com/7619656/208839010-6b7cce9c-6ab4-48ca-b854-12ed403efac2.png)

**Reply by kkthxbye-code** [reply]
Could you please try in another browser? Chrome/Chromium would be fine.

**Reply by Sivolen** [reply]
> Could you please try in another browser? Chrome/Chromium would be fine.

i tested on linux and window 10-11, chrome browser and firefox latest version

**Reply by kkthxbye-code** [reply]
This is caused by the fix to this issue: https://github.com/netbox-community/netbox/issues/11189

Enabling LocaleMiddleware has caused the output of floats to change for some languages, in your case it outputs a comma instead of a dot. This breaks rendering of the rack space progressbar.

I'll create an issue to track this.

As a workaround you can set your browser language to english.

**Reply by Sivolen** [reply]
> This is caused by the fix to this issue: https://github.com/netbox-community/netbox/issues/11189
> 
> 
> 
> Enabling LocaleMiddleware has caused the output of floats to change for some languages, in your case it outputs a comma instead of a dot. This breaks rendering of the rack space progressbar.
> 
> 
> 
> I'll create an issue to track this.
> 
> 
> 
> As a workaround you can set your browser language to english.

Ok, thanks
